src/NzbDrone.Core/Indexers/IndexerBase.cs

```diff
@@ -14,6 +17,8 @@ namespace NzbDrone.Core.Indexers
     public abstract class IndexerBase<TSettings> : IIndexer
         where TSettings : IIndexerSettings, new()
     {
+        private static readonly Regex MultiRegex = new (@"\b(?<multi>multi)\b", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
```suggestion
        private static readonly Regex MultiRegex = new (@"\b(?<multi>multi)\(?:\b|\d)", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```
https://trash-guides.info/Radarr/Radarr-collection-of-custom-formats/#multi

Reply:
What case does this catch?

Reply:
@TRaSH- background on the /d in multi cf? Looks like it's original

Reply:
Closing, they don't have cases that can be produced of this being needed